opentech/public/utils/context_processors.py

```diff
@@ -1,12 +1,16 @@
+from django.conf import settings
+
 from opentech.apply.home.models import ApplyHomePage
 from opentech.public.home.models import HomePage
-
 from opentech.public.mailchimp.forms import NewsletterForm
 
 
 def global_vars(request):
     return {
         'APPLY_SITE': ApplyHomePage.objects.first().get_site(),
         'PUBLIC_SITE': HomePage.objects.first().get_site(),
-        'newsletter_form': NewsletterForm()
+        'newsletter_form': NewsletterForm(),
+        'ORG_LONG_NAME': settings.ORG_LONG_NAME,
+        'ORG_SHORT_NAME': settings.ORG_SHORT_NAME,
+        'ORG_EMAIL': settings.ORG_EMAIL,
```

Comment:
@Parbhat Is this a decent way to make settings available in all templates?

Reply:
@frjo Wagtail has a [Site settings](https://docs.wagtail.io/en/v2.6.1/reference/contrib/settings.html) app specifically for this purpose. It will allow admins to change values from Wagtail admin. This method is also good as an organisation name, email etc. not change very often.